### PR TITLE
fix: Verification icons dark mode

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

### Issues

Verification icons (Proteus and MLS) were not adopted for Dark Mode.

### Causes (Optional)

Just wasn't implemented

### Solutions

Implement it

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt [hunk 1]
```diff
@@ -192,7 +192,6 @@ private fun WireDialogContent(
                 Text(
                     text = title,
                     style = MaterialTheme.wireTypography.title02,
-                    modifier = Modifier.weight(1f)
                 )
```
Comment (borichellow) [inline app/src/main/kotlin/com/wire/android/ui/common/WireDialog.kt:195]: Caused title ignoring `centerContent`

**Comment by AndroidBob** [comment]
Build [2524](https://10.10.124.134/job/AR-build-pipeline/2524/) **failed**.

**Comment by AndroidBob** [comment]
Build [2529](https://10.10.124.134/job/AR-build-pipeline/2529/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8609341)(8609341)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8609341)(8609341)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/common/VerifiedIcons.kt [hunk 2]
```diff
@@ -71,9 +73,11 @@ fun ProteusVerifiedIcon(
     modifier: Modifier = Modifier,
     @StringRes contentDescriptionId: Int = R.string.label_client_verified
 ) {
+    val iconId = if (isSystemInDarkTheme()) R.drawable.ic_certificate_valid_proteus_dark
+    else R.drawable.ic_certificate_valid_proteus
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/common/VerifiedIcons.kt]: question: wouldn't just putting icon for dark theme inside `drawable-night` directory work the same but without the need to create these `if`s and separate fields for `iconResId` and `darkIconResId`?

Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/ui/common/VerifiedIcons.kt]: dummy mistake 😅

**Comment by AndroidBob** [comment]
Build [2562](https://10.10.124.134/job/AR-build-pipeline/2562/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8642767)(8642767)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8642767)(8642767)-staging-compat.apk)

**Review by saleniuk — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2576](https://10.10.124.134/job/AR-build-pipeline/2576/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8643527)(8643527)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8643527)(8643527)-staging-compat.apk)

**Review by mchenani — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2585](https://10.10.124.134/job/AR-build-pipeline/2585/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.6.0(8645012)(8645012)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.6.0(8645012)(8645012)-staging-compat.apk)

**Pull request merged**